bundle/edu.gemini.model.p1/src/main/xsd/ProposalClass.xsd

```diff
@@ -129,6 +130,21 @@
     </xsd:complexType>
 
     <!--
+        Subaru Intensive Program
+    -->
+    <xsd:complexType name="SubaruIntensiveProgramClass">
+        <xsd:complexContent>
+            <xsd:extension base="ProposalClass">
+                <xsd:sequence>
+                    <xsd:element name="telescope" type="ExchangeTelescope"/>
+                    <xsd:element name="submission" type="SubaruIntensiveProgramSubmission"/>
+                </xsd:sequence>
+                <xsd:attribute name="tooOption" type="TooOption" use="optional"/>
```

Comment:
I read the information in REL-3193 but I'm still not sure what this is all about.  Are these LP programs for Gemini or are the observations at Subaru or maybe both?